yarn.lock

```diff
@@ -3473,6 +3496,15 @@
     color "^3.1.3"
     warn-once "^0.1.0"
 
+"@react-navigation/stack@^6.2.2":
```

Comment:
we have two version of `@react-navigation/stack` now, please unify versions to just one same everywhere

Reply:
Fixed